app/models/wakes/location.rb

```diff
@@ -13,7 +14,8 @@ def label
     path
   end
 
-  def url(protocol: 'http', host: ENV['DEFAULT_HOST'])
+  def url(protocol: 'http', host: nil)
+    host ||= self.host || ENV['DEFAULT_HOST']
```

Comment:
@gaganawhad should they? Does it make sense for wakes to have a different default host depending on environment?